docs/docs/methods.md

```diff
@@ -1614,3 +1614,63 @@ gte(1, 2)  // false
 gte(2, 2)  // true
 gte(3, 2)  // true
 ```
+
+## `has`
+
+Checks if an object has provided own property
+
+```js
+import { has } from 'nanoutils'
+
+const obj = {
+  a: 1,
+  __proto__: {
+    b: 2
+  }
+}
+
+has('a', obj)   // true
+has('b', obj)   // false
+```
+
+::: tip
+By own it means that `prototype` is not used to find a property
+:::
+
+## `hasIn`
+
+```js
+import { hasIn } from 'nanoutils'
+
+const obj = {
+  a: 1,
+  __proto__: {
+    b: 2
+  }
+}
+
+hasIn('a', obj)   // true
+hasIn('b', obj)   // true
+```
+
+::: tip
+Unlike [`has`](#has), `hasIn` uses `prototype` to find a property
+:::
+
+## `head`
+
+Returns first element of an `array` of `array`-ish `object` or `arguments`
```

Comment:
Some proposals here:
1. Method could also return first char of a string too.
2. 'array-ish' sounds weird in my opinion. I mean this `-ish` ending usually used in other cases (engl**ish**, fool**ish**, etc.) than this. I suggest using `like` _(array-like)_
3. There is no commas here at all. `... of array, array-like object or arguments list`
4. We could omit mentioning of `arguments`, because it's not a description, it's example of using. "Returns first element of a list or a string. Also, it could return object's value under key '0'"

Reply:
*-ish means something like that :)
generally, you're right